package.json

```diff
@@ -1384,6 +1384,254 @@
+        }
+      },
+      "@ctrl/shared-torrent": {
+        "6.3.2": {
+          "version": "6.3.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "@ctrl/tinycolor": {
+        "4.1.1": {
+          "version": "4.1.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        },
+        "4.1.2": {
+          "version": "4.1.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "@ctrl/torrent-file": {
+        "4.1.2": {
+          "version": "4.1.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "@ctrl/transmission": {
+        "7.3.1": {
+          "version": "7.3.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "@ctrl/ts-base32": {
+        "4.0.2": {
+          "version": "4.0.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "encounter-playground": {
+        "0.0.5": {
+          "version": "0.0.4",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "json-rules-engine-simplified": {
+        "0.2.1": {
+          "version": "0.2.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        },
+        "0.2.2": {
+          "version": "0.2.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        },
+        "0.2.3": {
+          "version": "0.2.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        },
+        "0.2.4": {
+          "version": "0.2.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "koa2-swagger-ui": {
+        "5.11.1": {
+          "version": "5.11.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        },
+        "5.11.2": {
+          "version": "5.11.0",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "@nativescript-community/gesturehandler": {
+        "2.0.35": {
+          "version": "2.0.34",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "@nativescript-community/sentry": {
+        "4.6.43": {
+          "version": "4.6.42",
+          "reason": "https://socket.dev/blog/tinycolor-supply-chain-attack-affects-40-packages"
+        }
+      },
+      "@nativescript-community/text": {
+        "1.6.13": {
+          "version": "1.6.9",
```

Comment:
should be 1.6.8 https://www.npmjs.com/package/@nativescript-community/text?activeTab=versions

1.6.9 not exists